COMPATIBILITY.md

```diff
@@ -0,0 +1,39 @@
+## Public API
+
+"Public API" means all public definitions in the artifacts under the
+`org.scalameta` Maven groupId whose fully-qualified name does not include
+`internal` and `contrib`, e.g. `scala.meta.Tree` or `org.langmeta.Position`:
```

Comment:
missing `org.scalameta` package, which is mentioned as private api below.

Reply:
Not sure what you mean by "below".